packages/nextjs/pages/index.tsx

```diff
@@ -1,64 +1,23 @@
-import Link from "next/link";
+import Image from "next/image";
+import logo from "../public/logo.svg";
 import { MetaHeader } from "@/components/MetaHeader";
 import type { NextPage } from "next";
-import { BugAntIcon, MagnifyingGlassIcon, SparklesIcon } from "@heroicons/react/24/outline";
 
 const Home: NextPage = () => {
   return (
-    <>
+    <div className="m-auto grid items-center">
       <MetaHeader />
-      <div className="flex items-center flex-col flex-grow pt-10">
-        <div className="px-5">
-          <h1 className="text-center mb-8">
-            <span className="block text-2xl mb-2">Welcome to</span>
-            <span className="block text-4xl font-bold">Scaffold-ETH 2</span>
-          </h1>
-          <p className="text-center text-lg">
-            Get started by editing{" "}
-            <code className="italic bg-base-300 text-base font-bold">packages/nextjs/pages/index.tsx</code>
-          </p>
-          <p className="text-center text-lg">
-            Edit your smart contract <code className="italic bg-base-300 text-base font-bold">YourContract.sol</code> in{" "}
-            <code className="italic bg-base-300 text-base font-bold">packages/hardhat/contracts</code>
-          </p>
+      <div className="bg-bt/60 w-screen grid xl:grid-cols-2 text-center py-5">
+        <div className="mx-auto grid xl:flex items-center text-l1 text-6xl xl:text-8xl font-kum font-bold gap-5 pb-5 xl:pb-0 px-10">
+          <Image src={logo} alt="Croissant Logo" className="mx-auto" />
```

Comment:
it is weird here that eslint or not throwing an error or warning about the `width` and `height` props of the Image component, doesn't it?

Reply:
oh, you are right. The width and height are defined in logo.svg, probably thats why eslint is ok with it